tests/continuous_gummyroll-test.ts

```diff
@@ -1,189 +0,0 @@
-import * as anchor from "@project-serum/anchor";
```

Comment:
Why rename this file?

Reply:
So that way it gets picked up in `Anchor.toml`

Reply:
Wait…

1. Doesn't this make it _not_ get picked up by `anchor test`? The pattern matches anything with `-test.ts` at the end.
2. If you really did intend to rename this file, make sure to use `git mv`. Git move keeps the blame history intact _through renames_. The way that you've done it here, you can no longer look up who wrote what, because the new file is 100% authored (and created) by your user.